deploy/src/main/deploy/docker/swarm_deploy.sh

```diff
@@ -78,7 +93,7 @@ echo Deploying Qpid Dispatch Router ...
 docker secret create -l project=$NS qdrouter-key.pem $CERTS/qdrouter-key.pem
 docker secret create -l project=$NS qdrouter-cert.pem $CERTS/qdrouter-cert.pem
 docker secret create -l project=$NS qdrouterd.json $SCRIPTPATH/qpid/qdrouterd-with-broker.json
-docker service create $CREATE_OPTIONS --name hono-dispatch-router -p 15671:5671 -p 15672:5672 \
+docker service create $CREATE_OPTIONS --name hono-dispatch-router -p 15671:5671 -p 15672:5672 -p 15673:5673 \
```

Comment:
I would rather have this port exposed only if `--no-adapters` is set ...

Reply:
Sure.